src/routes/profile.route.ts

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import { getProfile } from '../controllers/profile.controller';
+
+const profileRouter = express.Router();
+
+profileRouter.get('/me', getProfile);
```

Comment:
Need to use the passport authentication

```suggestion
profileRouter.get('/profile',  passport.authenticate('jwt', { session: false }), getProfile);
```